.github/scripts/generate_pr_description_zhipu.py

```
@@ -0,0 +1,81 @@
+                {"role": "system", "content": "You are an assistant helping to generate GitHub Pull Request descriptions based on code changes."},
+                {"role": "user", "content": prompt}
+            ],
+            temperature=0.3,
+            max_tokens=1000,
+        )
+
+        ai_output = response.choices[0].message.content.strip()
+        return ai_output
+
+    except FileNotFoundError:
+        logger.error(f"Prompt template file '{prompt_template_path}' not found.")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)